contentcuration/contentcuration/views/internal.py

```diff
@@ -643,9 +643,10 @@ def create_node(node_data, parent_node, sort_order):  # noqa: C901
         tag_data = node_data["tags"]
         if tag_data is not None:
             for tag in tag_data:
-                tags.append(
-                    ContentTag.objects.get_or_create(tag_name=tag, channel=channel)[0]
-                )
+                if len(tag) <= 30:
```

Comment:
Sorry I didn't clarify this, but if the tag is longer than 30 characters, lets make this stricter by raising a validation error. It would likely resemble a bug if this were to occur. 

I don't see an example of raising a validation error like this for this ricecooker API but perhaps we should define a specific validation error and catch it in `api_add_nodes_to_tree` where this function is called.